api/web3server.go

```diff
@@ -372,29 +377,19 @@ func (svr *Web3Server) getTransactionCount(in interface{}) (interface{}, error)
 	if err != nil {
 		return nil, err
 	}
-	num, err := svr.parseBlockNumber(blkNum)
-	if err != nil {
-		return nil, err
+	if blkNum == pendingBlockNumber {
+		pendingNonce, err := svr.coreService.ap.GetPendingNonce(ioAddr.String())
```

Comment:
does `GetPendingNonce` have test in /actpool?
if not, add a test here to verify it works as expected

Reply:
yes